src/BitOasis/Bitfinex/ExchangeRate/RabbitMqExchangeRateUpdater.php

```diff
@@ -48,14 +52,13 @@ public function addExchangeRateUpdateListener(ExchangeRateUpdateListener $listen
 	}
 
 	public function initializeMq(Channel $channel): PromiseInterface {
-		return $channel->exchangeDeclare(self::WEB_RPC_EXCHANGE, self::WEB_RPC_EXCHANGE_TYPE, false, true)
+		return $channel->exchangeDeclare(self::FIAT_RATES_EXCHANGE, self::FIAT_RATES_EXCHANGE_TYPE, false, true)
 			->then(function() use($channel) {
-				$queues = [];
-				$queues[] = $channel->queueDeclare('usd-exchange-rate.update', false, true);
-				return Promise\all($queues);
-			})->then(function() use($channel) {
+				return $channel->queueDeclare('', false, true, true);
```

Comment:
Why keeping it durable?

Reply:
Didn't think about it. Agree, transient also will work fine for current task :)